Mage.Sets/src/mage/cards/a/AshnodFleshMechanist.java

```diff
@@ -0,0 +1,76 @@
+package mage.cards.a;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.AttacksTriggeredAbility;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.costs.common.ExileFromGraveCost;
+import mage.abilities.costs.common.SacrificeTargetCost;
+import mage.abilities.costs.mana.GenericManaCost;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.abilities.effects.common.DoIfCostPaid;
+import mage.abilities.keyword.DeathtouchAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterControlledCreaturePermanent;
+import mage.filter.common.FilterControlledPermanent;
+import mage.filter.predicate.mageobject.AnotherPredicate;
+import mage.game.permanent.token.AshnodZombieToken;
+import mage.game.permanent.token.PowerstoneToken;
+import mage.target.common.TargetCardInYourGraveyard;
+import mage.target.common.TargetControlledPermanent;
+
+import java.util.UUID;
+
+/**
+ * @author PurpleCrowbar
+ */
+public final class AshnodFleshMechanist extends CardImpl {
+
+    private static final FilterControlledPermanent filter
+            = new FilterControlledCreaturePermanent("another creature");
```

Comment:
There's a static filter you can use here (common for these sacrifice effects).  `StaticFilters.FILTER_CONTROLLED_ANOTHER_CREATURE`

Reply:
Thanks, fixed for this card and all others I could find using that code